### PR TITLE
Fix XML namespace and others

### PR DESCRIPTION
**Pull request by yangbin**

### CODE REVIEW

**Comment by zz85** [comment]
lgtm :+1:

**Comment by jieyanhuang** [comment]
:+1:

File: jx/build/JXML.js [hunk 1]
```diff
@@ -186,10 +186,10 @@ JXML.prototype.generateJS = function(build_assets) {
  * TODO: text interleaved with elements not handled well
  */
 function XMLToJSON(element) {
-	var name = (element.namespaceURI? element.namespaceURI + '/' : '') + element.tagName,
+	var name = (element.namespaceURI? element.namespaceURI + '/' : '') + element.localName,
 		children = {},
 		childNodes = element.childNodes,
-		children_index = 1,
+		children_index = 0,
```
Comment (tgohn) [inline jx/build/JXML.js]: actually we thought wrong this afternoon. There were no problem with this code :p
``` javascript
a = 1; 
console.log(a++)  // 1
console.log(a)  // 2
```

Comment (yangbin) [inline jx/build/JXML.js]: that's what I thought. so where did the 2 come from?

**Comment by yangbin** [comment]
:haircut:

File: jx/build/JXML.js (hunk 1 above)
Comment (tgohn) [inline jx/build/JXML.js]: could be from the omitted empty text `"  "`

**Comment by tgohn** [comment]
:+1:  beside the index changed to 0

**Pull request merged**